src/sentry/shared_integrations/exceptions/__init__.py

```diff
@@ -171,8 +171,11 @@ class DuplicateDisplayNameError(IntegrationError):
 
 class IntegrationFormError(IntegrationError):
     def __init__(self, field_errors: Mapping[str, Any]) -> None:
-        super().__init__("Invalid integration action")
-        self.field_errors = field_errors
+        error = "Invalid integration action"
+        if field_errors:
+            error = str(field_errors)
```

Comment:
Does map to str always come in an ok format?

Reply:
this should be fine, i just tested it in a python playground